Drivers/Levoit/VeSyncIntegration.groovy

```diff
@@ -173,21 +172,11 @@ def Boolean updateDevices()
             def configModule = e.value
 
             if (dni.endsWith("-nl")) continue;
-            
+
             logDebug "Updating ${dni}"
 
             def dev = getChildDevice(dni)
-
-            sendBypassRequest(dev, command) { resp ->
-                if (checkHttpResponse("update", resp))
-                {
-                    def status = resp.data.result
-                    if (status == null)
-                        logError "No status returned from getPurifierStatus: ${resp.msg}"
-                    else
-                        result = dev.update(status, getChildDevice(dni+"-nl"))
-                }
-            }
+            dev.update()
```

Comment:
Interesting -- can it be reduced to just that?

Reply:
Hmmmm, I just read over your driver for LevoitCore200S Light.groovy

It seems that you created a separate driver for just the night light? Any thoughts on just changing that as a method on the drivers? i.e. in LevoitCore200S and LevoitCore300S just have a setNightLight method.

The reason is, ideally the `update` method should be delegated to the child device, because the Humidifier calls `getHumidifierStatus` and not `getPurifierStatus` like all the existing drivers